src/main/java/hudson/tasks/junit/TestResultAction.java

```diff
@@ -86,8 +88,14 @@ public TestResultAction(TestResult result, BuildListener listener) {
         this((Run) null, result, listener);
     }
 
+    @SuppressWarnings("deprecation")
```

Comment:
Does it really make sense to filter away deprecations? It would be better to leave them as warnings